### DISCUSSION TITLE
Print on Demand API Connections

### DISCUSSION THREAD

**Discussion by hodlrodeo** [PlebeianTech/plebeian.market#357] [Ideas]
The ability to set up a print on demand site on Nostr unlocks a growing market and novel interactions.

https://www.printful.com/api

https://www.printful.com/blog/print-on-demand-statistics

1. The print-on-demand industry is growing at a compound annual growth rate (CAGR) of 25.3%
2. The US is the largest market for print-on-demand
3. The growing demand for custom and unique products is a vital driver of the POD market 
4. 25% of consumers say they’d spend more money if the product is sustainable
5. The eco-friendly clothing segment will reach $10.1 billion in global annual sales by 2025
6. The average monthly search volume for “print-on-demand” is 132.7K searches
7. The top trending print-on-demand categories are apparel, home decor, drinkware, and accessories
8. The top trending apparel print-on-demand products are t-shirts, sportswear, and leggings

**Comment by hodlrodeo** [comment]
Currently users must have their own inventory and shipping methods in place to use this marketplace. Connecting a print on demand service removes this barrier while removing the need for current print on demand users to pay for an Ecommerce website like Shopify or squareup.

**Reply by ChiefmonkeyArt** [reply]
Please do create an issue for this... another great idea and one that someone may create for our plugin/app market if we don't don't first 👍

**Reply by hodlrodeo** [reply]
Done 🤙

https://github.com/PlebeianApp/plebeian.market/issues/522
